### PR TITLE
fix(dev): upgrade script when using scope

### PR DESCRIPTION
**Pull request by mkilpatrick**

### CODE REVIEW

File: packages/pages/src/upgrade/upgrade.ts [hunk 1]
```diff
@@ -23,23 +21,32 @@ interface UpgradeArgs {
...
-  const source = path.resolve(root, "src");
   const projectStructure = await ProjectStructure.init(scoped);
-  await updateDevDependencies(root);
-  checkLegacyMarkdown(source);
-  removeFetchImport(source);
-  updatePackageScripts(root);
-  updatePackageEngines(root);
+  const source = path.resolve(projectStructure.config.rootFolders.source);
+
+  // check proper Node version first
```
Comment (asanehisa) [inline packages/pages/src/upgrade/upgrade.ts:27]: little random these all start lowercase?

**Review by asanehisa — APPROVED** [review]
lgtm

File: packages/pages/src/upgrade/upgrade.ts [hunk 2]
```diff
@@ -23,23 +21,32 @@ interface UpgradeArgs {
...
+  checkLegacyMarkdown(source);
+
+  // install deps
+  await installDependencies();
+
+  // migrate to consolidated configs (for all scopes)
   if (!args.noMigration) {
     await migrateConfigs(projectStructure);
   }
-  await templatesHandler(scoped);
```
Comment (jwartofsky-yext) [inline packages/pages/src/upgrade/upgrade.ts:41]: Do we want to remove these?

**Review by jwartofsky-yext — APPROVED** [review]
lgtm

File: packages/pages/src/upgrade/upgrade.ts (hunk 1 above)
Comment (mkilpatrick) [reply packages/pages/src/upgrade/upgrade.ts:27]: ack

**Pull request merged**